xontrib/gitstatus.xsh

```diff
@@ -0,0 +1,140 @@
+'''informative git status for prompt'''
+
+import re
+import os
+import builtins
+import functools
+import subprocess
+
+__all__ = []
+
+_env = builtins.__xonsh_env__
+
+check_output = functools.partial(subprocess.check_output,
+                                 env=_env.detype(),
```

Comment:
I don't think this is valid at the global scope.  The env needs to be detyped each time, right before the call